papis/__init__.py

```diff
@@ -1,20 +1,4 @@
-import os
-
-# Information
 __license__ = "GPLv3"
 __version__ = "0.12"
 __author__ = __maintainer__ = "Alejandro Gallo"
 __email__ = "aamsgallo@gmail.com"
-
-
-if os.environ.get("PAPIS_DEBUG"):
```

Comment:
Was this used for anything? I might have been a bit hasty in removing it..

My understanding was that the logging gets initialized in `papis.commands.default`, which has flags to enable more verbosity, so it seemed redundant.

Reply:
this was as far as I remember to see the timing and debugging information of importing modules in papis before the default command kicks in, in order to optimize then startup time. but I think this should rather be done better by a profiling tool, so I guess it's ok to remove?